common/predictive-text/worker/model-compositor.ts

```diff
@@ -341,6 +341,12 @@ class ModelCompositor {
           // A predictive text default (on iOS, at least) - immediately wordbreak 
           // on suggestions accepted mid-word.
           suggestion.transform.insert += punctuation.insertAfterWord;
+
+          // // Unfortunately, we can't quite enable the next two lines in iOS yet; there's an
+          // // odd issue with its textDocumentProxy that gives us the wrong results.
+          // 
+          // let righthandSplit = tokenization.right[0];
+          // suggestion.transform.deleteRight = (suggestion.transform.deleteRight || 0) + righthandSplit.kmwLength();
```

Comment:
Note to future self:  when the final two lines are enabled, delete the first three lines.